streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -53,7 +52,20 @@
         FROM_100_TO_23
     }
 
+    // Temporarily moved from StreamsConfig to here to hide the built-in metrics version config
```

Comment:
I think we can remove this line since it would be permanent for 2.4 right?

Reply:
Yes, you are right!